jquery.extend/jquery.extend.d.ts

```diff
@@ -0,0 +1,354 @@
+// Type definitions for jQuery 1.10.x / 2.0.x
+// Project: http://jquery.com/
+// Definitions by: falsandtru, jquery.d.ts definitors
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/* *****************************************************************************
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
```

Comment:
this project is under MIT license. https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE